Directory.Build.props

```diff
@@ -5,6 +5,7 @@
 
   <Import Project="version.props" />
   <Import Project="build\dependencies.props" />
+  <Import Condition="'$(ANCMAlphaBuild)' != ''" Project="build\dependencies_alpha.props" />
```

Comment:
ANCMAlphaBuild -> AncmOobBuild?

I assume you intend to get past alpha at some point.